website/client/js/services/achievementServices.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Services that handle achievement logic.
+ */
+
+angular.module('habitrpg').factory('Achievement',
+['$rootScope', function($rootScope) {
+
+  function displayAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl', size:'sm'});
+  }
+
+  function displayBulkyAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl'});
```

Comment:
In the example modal buttons in the documentation, the first modal is with no size param, which is what the larger achievement modals originally used.